stories/utils/analytics.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Analytics module utility functions.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import * as fixtures from '../../assets/js/modules/analytics/datastore/__fixtures__';
+import { STORE_NAME } from '../../assets/js/modules/analytics/datastore/constants';
+import { STORE_NAME as CORE_SITE, AMP_MODE_SECONDARY } from '../../assets/js/googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_MODULES } from '../../assets/js/googlesitekit/modules/datastore/constants';
+import { withActive } from '../../assets/js/googlesitekit/modules/datastore/__fixtures__';
+import { createBuildAndReceivers } from '../../assets/js/modules/tagmanager/datastore/__factories__/utils';
+
+/**
+ * Generates a story for a case when a GTM with Analytics property ID is already connected.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object}      args                Story arguments.
+ * @param {WPComponent} args.Component      Story component.
+ * @param {boolean}     args.useExistingTag Whether to use an existing tag or not.
+ * @param {boolean}     args.gaPermission   Whether the current user has GA tag permissions.
+ * @param {boolean}     args.gtmPermission  Whether the current user has GTM tag permissions.
+ * @return {Function} Story callback function.
+ */
+export function generateGTMPropertyStory( {
```

Comment:
Maybe call this function more specifically, like `generateGTMAnalyticsPropertyStory`? And I think the filename should match the function name and only contain that, i.e. `generate-gtm-analytics-property-story.js`.

Reply:
Yep, updated.